### ISSUE TITLE
Can't start latest Linux build (dev-c837a06)

### ISSUE THREAD

**Issue by mrvanes** [alexrj/Slic3r#3988]
### Version
dev-c837a06

When calling the included Slic3r script with --gui, I get the following output:
```
Can't locate loadable object for module Wx::AUI in @INC (@INC contains: ...
```
Where the "contains:" points to all the relevant inc dirs beneath the Slic3r install dir. Tried to compare libs and modules to last working dev build, but couldn't find any significant differences that might account for the problem.

**Comment by lordofhyphens** [comment]
Ah, same bug as #3986

**Comment by mrvanes** [comment]
I confirm this is fixed in dev-6a7057b for me. Thx!
